### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by benchan777**
- improved overall aesthetics of the website
- fixed an issue where ajax wasn't receiving a response and prevented new page load/page redirects

### CODE REVIEW

**Comment by benchan777** [comment]
Significantly more changes made since original pull request

- Fixed site background image stretching/tiling improperly depending on page length and window size
- Source link in navbar linked to wrong page. Fixed that
- Updated playlist_details page to be able to display retrieved data from backend. Still needs refining. Current data are just placeholders so we can easily see how to access the data properly from the JSON
- Updated Playlist database model to store accessToken and refreshToken
- Add GET route for playlist_details that retrieves information about the user's most newly created playlist. If there are multiple playlists, sorts to only return information about latest playlist

File: public/styles.css [hunk 1]
```diff
@@ -1,23 +1,68 @@
...
     max-height: 500px;
     font-size: 16px;
 
 }
 
 .ui-menu .ui-menu-item{
     margin: none;
-    padding-left: 5px;
-    padding-right:5px;
+    padding-left: 0px;
```
Comment (numel007) [inline public/styles.css:64]: Bring back the padding plz

File: src/controllers/routes.js [hunk 2]
```diff
@@ -70,6 +71,30 @@ router.post("/create_playlist", middleware.verifyToken, (req, res) => {
 	}
 });
 
+router.get("/playlist_details", middleware.verifyToken, (req, res) => {
+	if (req.noRefreshToken) {
+		res.redirect("/login");
+	} else {
+		Playlist.findOne({ refreshToken: req.cookies.refreshToken}).sort({ _id: -1 })
+		.then( results => {
+			console.log(results.playlistId)
```
Comment (numel007) [inline src/controllers/routes.js]: remove `console.log(results.playlistId)`

**Review by numel007 — CHANGES_REQUESTED** [review]

File: public/styles.css (hunk 1 above)
Comment (benchan777) [reply public/styles.css:64]: No.

File: src/controllers/routes.js (hunk 2 above)
Comment (numel007) [reply src/controllers/routes.js]: Fixed in [this commit.](https://github.com/numel007/SPG-nodejs/pull/16/commits/977270f9bcf9671ee8abe3f91b6b13f19e8b1281).

**Review by numel007 — APPROVED** [review]

**Comment by numel007** [comment]
LGTM!

**Pull request merged**